docker/Dockerfile.base

```diff
@@ -19,6 +19,8 @@ RUN apt-get update && \
       knot-dnsutils \
       netcat \
       tcpdump \
+      conntrack \
+      bsdmainutils \
```

Comment:
for debugging iptables issues

Reply:
am I reading this right this has no actual binaries? https://packages.debian.org/sid/all/bsdmainutils/filelist

Reply:
Ugh. hmm.. it works on ubuntu. I am installing this for hexdump. Nothing else.

Reply:
btw, does the base image get updated automatically if I update this file?